package.json

```diff
@@ -147,11 +148,10 @@
     "stylelint-config-styled-components": "^0.1.1",
     "stylelint-processor-styled-components": "^1.5.1",
     "ts-jest": "^22.0.2",
-    "ts-node": "^8.0.2",
     "tslint": "^5.11.0",
     "tslint-config-prettier": "^1.6.0",
     "tslint-plugin-relay": "^0.0.2",
-    "typescript": "^3.0.3",
+    "typescript": "^3.1.5",
```

Comment:
This bump was needed to support `resolveJsonModule` when targeting `es2015` output: https://github.com/Microsoft/TypeScript/pull/26825